liota/lib/utilities/utility.py

```diff
@@ -155,24 +182,61 @@ def _find_path(self):
     def get_liota_fullpath(self):
         return LiotaConfigPath.path_liota_config
 
+
 def read_liota_config(section, name):
-     """Returns the value of name within the specified section.
- """
-     config = ConfigParser.RawConfigParser()
-     fullPath = LiotaConfigPath().get_liota_fullpath()
-     if fullPath != '':
-         try:
-             if config.read(fullPath) != []:
-                 try:
-                     value = config.get(section, name)			
-                 except ConfigParser.ParsingError as err:
-                     log.error('Could not parse log config file')
-             else:
-                 raise IOError('Cannot open configuration file ' + fullPath)
-         except IOError as err:
-             log.error('Could not open log config file')
-     else:
-         # missing config file
-         log.warn('liota.conf file missing')
-     return value
- 
+    """Returns the value of name within the specified section.
+    """
+    config = ConfigParser.RawConfigParser()
+    fullPath = LiotaConfigPath().get_liota_fullpath()
+    if fullPath != '':
+        try:
+            if config.read(fullPath) != []:
+                try:
+                    value = config.get(section, name)
+                except ConfigParser.ParsingError as err:
+                    log.error('Could not parse log config file' + str(err))
+            else:
+                raise IOError('Cannot open configuration file ' + fullPath)
+        except IOError as err:
+            log.error('Could not open log config file')
+    else:
+        # missing config file
+        log.warn('liota.conf file missing')
+    return value
+
+
+class Credentials:
```

Comment:
EdgeSystemIdentity and RemoteSystemIdentity didn't suit well.  `Credentials` will be simply a data object (utility) that holds cert paths and credentials used in various classes.  Goal is to reduce number of constructor params of DccComms & DeviceComms related to Credentials.  So, we thought why to create a separate module for this and kept it in Utility itself.